src/main/java/org/isf/hospital/gui/HospitalBrowser.java

```diff
@@ -250,12 +250,12 @@ public void saveConfirm() {
 	public JPanel getJButtonPanel() {
 		if (jButtonPanel==null){
 			jButtonPanel= new JPanel();
-			editJButton = new JButton(MessageBundle.getMessage("angal.common.edit"));
-			editJButton.setMnemonic(KeyEvent.VK_E);
+			editJButton = new JButton(MessageBundle.getMessage("angal.common.edit.btn"));
+			editJButton.setMnemonic(MessageBundle.getMnemonic("angal.common.edit.btn.key"));
 			updateJButton = new JButton(MessageBundle.getMessage("angal.hospital.update"));
 			updateJButton.setMnemonic(KeyEvent.VK_U);
-			exitJButton = new JButton(MessageBundle.getMessage("angal.common.close"));
-			exitJButton.setMnemonic(KeyEvent.VK_C);
+			exitJButton = new JButton(MessageBundle.getMessage("angal.common.close.btn"));
+			exitJButton.setMnemonic(MessageBundle.getMnemonic("angal.common.close.btn.key"));
```

Comment:
this button maybe needs refactoring as well

Reply:
Thanks; done.